src/components/WagmiProvider.tsx

```diff
@@ -1,16 +1,54 @@
 'use client';
-import { WagmiConfig, createClient, configureChains } from 'wagmi';
-import { mainnet, arbitrum, arbitrumGoerli, goerli } from 'wagmi/chains';
+import { WagmiConfig, createClient, configureChains, Chain } from 'wagmi';
+import {
+  mainnet,
+  arbitrum,
+  arbitrumGoerli,
+  goerli,
+  localhost,
+} from 'wagmi/chains';
 import { publicProvider } from 'wagmi/providers/public';
 import { jsonRpcProvider } from 'wagmi/providers/jsonRpc';
 import { ReactNode } from 'react';
+import { ChainId, rpcURLs } from '@/utils/network';
+
+/**
+ * For e2e testing
+ */
+export const localL1Network: Chain = {
+  id: ChainId.Local,
+  name: 'EthLocal',
+  network: 'local',
```

Comment:
As far as I remember this should be `localhost` because that's Metamask's default local network. This way we won't have to add a new local network but use the one Metamask has instead.

Reply:
Updated